src/main/java/com/linecorp/armeria/client/HttpConfigurator.java

```diff
@@ -256,93 +267,19 @@ private Http2ConnectionHandler createHttp2ConnectionHandler() {
 
     private static final class ExtendedHttpToHttp2ConnectionHandler extends HttpToHttp2ConnectionHandler {
 
-        private final SessionListener sessionListener;
-
-        static final class Builder extends BuilderBase<HttpToHttp2ConnectionHandler, Builder> {
-            private SessionListener sessionListener;
-
-            public Builder sessionListener(SessionListener sessionListener) {
-                this.sessionListener = sessionListener;
-                return this;
-            }
-
-            @Override
-            protected HttpToHttp2ConnectionHandler build0(Http2ConnectionDecoder decoder,
-                                                          Http2ConnectionEncoder encoder) {
-                return new ExtendedHttpToHttp2ConnectionHandler(decoder, encoder, initialSettings(),
-                                                               isValidateHeaders(), sessionListener);
-            }
-        }
-
         ExtendedHttpToHttp2ConnectionHandler(Http2ConnectionDecoder decoder, Http2ConnectionEncoder encoder,
-                                             Http2Settings initialSettings, boolean validateHeaders,
-                                             SessionListener sessionListener) {
+                                             Http2Settings initialSettings, boolean validateHeaders) {
             super(decoder, encoder, initialSettings, validateHeaders);
-            this.sessionListener = sessionListener;
         }
 
         @Override
         public void close(ChannelHandlerContext ctx, ChannelPromise promise) throws Exception {
             //mark channel session inactive due to prevent reuse on channel pool
-            sessionListener.sessionDeactivated(ctx);
+            HttpSessionHandler.deactivate(ctx.channel());
 
             //TODO(inch772) : When https://github.com/netty/netty/issues/4210 has be resolved, remove this
             encoder().connection().forEachActiveStream(closeAllStreams);
```

Comment:
I as remember this line for force close stream id 1 when channel close by client side.
I tested channel can be closed even if without this line. 
How about remove also remove this line?